rust/src/treefile.rs

```diff
@@ -663,6 +664,15 @@ enum Include {
     Multiple(Vec<String>),
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+/// The database backend; see https://github.com/coreos/fedora-coreos-tracker/issues/609
+/// and https://fedoraproject.org/wiki/Changes/Sqlite_Rpmdb
+enum RpmdbBackend {
+    BDB,
+    Sqlite
```

Comment:
There are users trying to use rpm-ostree on openSUSE, which only offers `ndb` backend, could you please add support for that option too?

Reply:
Sure, done.